src/__tests__/plugin.js

```diff
@@ -65,4 +65,21 @@ describe('gatsby-remark-embedder', () => {
       "
     `);
   });
+
+  test('can transform a link with no label (emptylink)', async () => {
```

Comment:
We don't need an extra test, because we can just use the kitchensink test instead